src/routes/Detail.js

```diff
@@ -1,9 +1,51 @@
 import React from 'react';
+import RepositoryDetailModal from 'components/RepositoryDetailModal';
+import axios from 'axios';
 
 
 export default class Detail extends React.Component {
+  state = { isOpen: false };
```

Comment:
I think we don't need the state here. When `Detail` component is mounted (i.e. when `detail` route is active), the modal is just open